src/windows/utils.rs

```diff
@@ -0,0 +1,67 @@
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use std::path::PathBuf;
+use symbolic_debuginfo::pe::PeObject;
+
+use crate::utils;
+
+pub fn get_win_path(path: &str) -> PathBuf {
+    PathBuf::from(path.replace("\\", "/"))
+}
+
+fn try_to_find_pdb(path: PathBuf, pdb_filename: &str) -> Option<Vec<u8>> {
+    let pdb = path.with_file_name(pdb_filename);
+    if pdb.is_file() {
+        Some(utils::read_file(pdb))
+    } else {
+        let mut pdb = std::env::current_dir().expect("Unable to get the current working directory");
+        pdb.set_file_name(pdb_filename);
+        if pdb.is_file() {
+            Some(utils::read_file(pdb))
+        } else {
+            None
+        }
+    }
+}
+
+pub fn get_pe_pdb_buf<'a>(
+    path: &PathBuf,
+    buf: &'a [u8],
+) -> Option<(PeObject<'a>, Vec<u8>, String)> {
+    let path = PathBuf::from(path);
+    let pe = PeObject::parse(&buf)
+        .unwrap_or_else(|_| panic!("Unable to parse the PE file {}", path.to_str().unwrap()));
+    if let Some(pdb_filename) = pe.debug_file_name() {
+        let pdb_filename = pdb_filename.into_owned();
+
+        #[cfg(windows)]
+        let (pdb, pdb_name) = {
```

Comment:
Split this out into platform-specific functions like `win_try_to_find_pdb()` and `unix_try_to_find_pdb()`.

Reply:
done